tests/unit/lib/cookiecutter/questions.yaml

```diff
@@ -0,0 +1,22 @@
+{
+  "Questions": [
+    {
+      "key": "1st",
+      "question": "any text with variable X substitution to %(X)s",
+    },
+    {
+      "key": "2nd",
+      "question": "any text",
+      "options": ["option1", "option2"],
+      "nextQuestion": {
+        "option1": "1st"
+      }
+    },
+    {
+      "key": "3rd",
+      "question": "any text",
+      "isRequired": true,
+      "kind": "confirm"
+    }
+  ]
+}
```

Comment:
this test file is a JSON file, why it has ".yaml" as extension?

Reply:
good catch. Fixed

Reply:
Technically JSON is valid YAML :^)